pages/learn/develop/hardware.md

```diff
@@ -28,4 +28,6 @@ For more details on interacting with external hardware, check out these guides:
 - [i2C and SPI][i2c-spi]
 
 [gpio]:/learn/develop/hardware/gpio
-[i2c-spi]:/learn/develop/hardware/i2c-and-spi
\ No newline at end of file
+[i2c-spi]:/learn/develop/hardware/i2c-and-spi
+[base-image-wiki-link]:/runtime/resin-base-images/
```

Comment:
Should this be `/reference/base-images/resin-base-images` to match the link in dockerfile.md ?